app/page.tsx

```diff
@@ -211,80 +210,135 @@ export default function IndexPage() {
-              </li>
-            </ul>
-          </div>
-          <div className="p-5">
-            <ul>
-              <p className="select-none pb-3 text-2xl font-bold text-slate-900 dark:text-white">Partner</p>
-              <li className="text-md cursor-pointer pb-2 font-semibold text-gray-500">
-                Our Partners
-              </li>
-              <li className="text-md cursor-pointer pb-2 font-semibold text-gray-500">
-                Partner with us
-              </li>
-            </ul>
+            <div className="p-5">
+              <ul>
+                <p className="text-slate-900 dark:text-white font-normal text-xl pb-3 select-none">
+                  Contact
+                </p>
+                <li className="text-gray-500 text-base pb-2 font-normal cursor-pointer">
+                  Contact us
+                </li>
+              </ul>
+            </div>
+            <div className="p-5">
+              <ul>
+                <p className="text-slate-900 dark:text-white font-normal text-xl pb-3 select-none">
+                  Partner
+                </p>
+
+                <li className="text-gray-500 text-base pb-2 font-normal cursor-pointer">
+                  Partner with us
+                </li>
+              </ul>
+            </div>
           </div>
         </div>
-        <div className="flex flex-col items-center justify-center bg-white p-5  text-center dark:bg-background">
-          <h1 className=" select-none font-semibold text-slate-900 dark:text-white">
-            © 2023 All rights reserved | Build with ❤ by{" "}
-            <span className="cursor-pointer select-none font-semibold hover:text-blue-600">
-              WebXDAO{" "}
-            </span>
-          </h1>
+        <div className="flex gap-72 justify-center mt-20 ">
+          <div className=" flex flex-col  items-center text-center  pt-4  ">
+            <h1 className=" text-slate-900 dark:text-white font-medium select-none text-base">
+              © 2023 All rights reserved | Build with ❤ by{" "}
+              <span className="hover:text-blue-600  cursor-pointer select-none">WebXDAO </span>
+            </h1>
+          </div>
+          <div className=" text-slate-500 dark:text-white text-base font-medium pl-36">
+            <div className="flex  ">
+              <div className="pt-3  flex ">
+                Powered by
+                <img src="Footlogo1.png" alt="logo_image" className=" -mt-2" />
+                <img src="Footlogo2.png" alt="logo_image" className=" h-6 " />
+              </div>
+              <div className="p-2">
+                <a className=" hover:text-blue-600 px-8">Terms</a>
+                <a href="#" className="px-8 hover:text-blue-600">
+                  Privacy
+                </a>
+                <a href="#" className="px-8 hover:text-blue-600">
+                  Cookies
+                </a>
+              </div>
+            </div>
+          </div>
         </div>
```

Comment:
This footer look like this
![image](https://github.com/WebXDAO/WebXDAO.github.io/assets/32319439/8477892f-6451-4f9c-a0b9-f116018f1a80)